rules/integrations/aws/exfiltration_ec2_snapshot_change_activity.toml

```diff
@@ -23,7 +23,62 @@ interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
 name = "AWS EC2 Snapshot Activity"
-note = """## Config
+note = """## Triage and analysis
+
+### Investigating AWS EC2 Snapshot Activity
+
+Amazon EC2 snapshots are a mechanism to create point-in-time references to data that resides in the storage volume.
+System administrators commonly use this for backup operations and data recovery.
+
+This rule looks for the modification of snapshot attributes using the API `ModifySnapshotAttribute` action. This can be
+used to share the snapshot with unauthorized third parties, which means you are giving others access to all the data on
+the snapshot.
+
+#### Possible investigation steps
+
+- Identify the user account that performed the action and whether it should perform this kind of action.
```

Comment:
```suggestion
- Identify the user account that performed the action and whether it should perform this kind of action.
- Search for dry run attempts against the resource ID of the snapshot from other user accounts within CloudTrail.
```

Reply:
Without using the specific API request, you can dry run against a snapshot to determine if you have privileges or not prior to your API call. Since this does not actually send a request, the only way to find it would be via logs to that specific resource ID of the snapshot.